pkg/csilvm/server.go

```diff
@@ -1260,9 +1260,16 @@ func determineFilesystemType(devicePath string) (string, error) {
 }
 
 func formatDevice(devicePath, fstype string) error {
-	output, err := exec.Command("mkfs", "-t", fstype, devicePath).CombinedOutput()
-	if err != nil {
-		return errors.New("csilvm: formatDevice: " + string(output))
+	// scrub the first 256k of the device to head off any mkfs probe misfires.
+	output, err := exec.Command(
+		"dd", "if=/dev/zero", "of="+devicePath, "bs=512", "count=512", "conv=notrunc",
```

Comment:
TIL `conv=notrunc`

Reply:
I would expect it not to have an effect on device files (only regular files) but  didn't know it even existed, very cool

Reply:
i think it probably only does affect regular files. probably safe to remove at some point. it was part of my local testing because of how I was playing w/ regular files.